src/rest/connector/libs/viptela/implementation.py

```diff
@@ -135,6 +135,11 @@ def connect(self, timeout=30, port="8443", protocol='https'):
                                  timeout=timeout)
 
         if resp.status_code == 200:
+            # html login page gets returned if credentials are invalid
+            if resp.content and b'<html>' in resp.content:
```

Comment:
Seems to vague, can we be more precises, like look for login or something in the content

Reply:
`<p id="errorMessageBox" name="errorMessageBox" class='errorMessageBox '>Invalid User or Password</p>` this is part of the response. Updated for check for that